code/game/machinery/autolathe.dm

```diff
@@ -219,3 +219,36 @@
 			qdel(I)
 			return 1
 	return 0
+
+/obj/machinery/r_n_d/fabricator/mechanic_fab/autolathe/MouseDropTo(atom/over_object, mob/user)
+	if(istype(over_object, /obj/item/weapon/storage/bag/gadgets/part_replacer))
+		if(user.stat != CONSCIOUS)
```

Comment:
```suggestion
		if(user.incapacitated() || !user.Adjacent(src))
```
Remove the other checks.
There's also no need to get the turf just to pass it to `playsound`. `playsound` will do that for you.

Reply:
the get turf is also for the proximity check

Reply:
but now I see the Adjacent part, ahahah, sorry and thank you!